docs/cloud/managing-airbyte-cloud/dbt-cloud-integration.md

```diff
@@ -1,4 +1,6 @@
-# Using the dbt Cloud integration 
+# Utilize the dbt Cloud integration 
```

Comment:
Should I change "Utilize" to "Use" or another verb?

Reply:
yep, utilize > use